### PR TITLE
Command templates for mqtt fan

### PR DESCRIPTION
**Pull request by jbouwh**
## Proposed change

Added support for command templates

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [x] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/48413
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_docs/mqtt/discovery.markdown [hunk 1]
```diff
@@ -45,8 +45,6 @@ discovery_prefix:
   type: string
 {% endconfiguration %}
 
-### Discovery topic
```
Comment (emontnemery) [inline source/_docs/mqtt/discovery.markdown]: This is not right, please revert this change.

File: source/_docs/mqtt/discovery.markdown [hunk 2]
```diff
@@ -57,11 +55,9 @@ The discovery topic need to follow a specific format:
 - `<node_id>` (*Optional*):  ID of the node providing the topic, this is not used by Home Assistant but may be used to structure the MQTT topic. The ID of the node must only consist of characters from the character class `[a-zA-Z0-9_-]` (alphanumerics, underscore and hyphen).
 - `<object_id>`: The ID of the device. This is only to allow for separate topics for each device and is not used for the `entity_id`. The ID of the device must only consist of characters from the character class `[a-zA-Z0-9_-]` (alphanumerics, underscore and hyphen).
 
-Best practice for entities with a `unique_id` is to set `<object_id>` to `unique_id` and omit the `<node_id>`.
```
Comment (emontnemery) [inline source/_docs/mqtt/discovery.markdown]: This is not right, please revert this change.

File: source/_docs/mqtt/discovery.markdown [hunk 3]
```diff
@@ -57,11 +55,9 @@ The discovery topic need to follow a specific format:
...
-
 The payload must be a JSON dictionary and will be checked like an entry in your `configuration.yaml` file if a new device is added. This means that missing variables will be filled with the platform's default values. All configuration variables which are *required* must be present in the initial payload send to `/config`.
 
-Subsequent messages on a topic where a valid payload has been received will be handled as a configuration update, and a configuration update with an empty payload will cause a previously discovered device to be deleted.
```
Comment (emontnemery) [inline source/_docs/mqtt/discovery.markdown]: This is not right, please revert this change.

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks, @jbouwh 👍
